components/edit-collective/sections/Host.js

```diff
@@ -172,17 +171,16 @@ class Host extends React.Component {
                     />
                   </p>
                   <p>
-                    <Button
-                      bsStyle="primary"
-                      type="submit"
```

Comment:
It's good to keep the `type="submit"` if already set, makes things explicit in the DOM